python/tvm/contrib/download.py

```diff
@@ -59,7 +59,7 @@ def download(url, path, overwrite=False, size_compare=False, verbose=1, retries=
     # Stateful start time
     start_time = time.time()
     dirpath = os.path.dirname(path)
-    if not os.path.isdir(dirpath):
+    if not os.path.isdir(dirpath) and not dirpath is '':
```

Comment:
You can use `not dirpath` to check whether a string is empty.
Could you also put the non-empty check before the `os.path.isdir` check?

Reply:
Changed as desired. There are other ways to [check if empty](https://stackoverflow.com/questions/9573244/most-elegant-way-to-check-if-the-string-is-empty-in-python) too.